stdlib.sh

```diff
@@ -1202,9 +1211,14 @@ on_git_branch() {
     return 1
   elif [ -z "$1" ]; then
     return 0
+  elif [ "$1" = "-r" ] && [ -z "$2" ]; then
```

Comment:
```suggestion
  elif [[ "$1" = "-r" && -z "$2" ]]; then
```
Let's use bash tests everywhere.

Reply:
Done!